iconlogger/colors.go

```diff
@@ -0,0 +1,44 @@
+package iconlogger
+
+import (
+	"io"
+
+	"github.com/kubescape/go-logger/helpers"
+
+	"github.com/fatih/color"
```

Comment:
not the new library?

Reply:
I thought of doing that, but then thought It would be better to remove the fatih/color library incremently and not conflict with other PRs.

I'd rather open a single PR which completely removes all occurences of the old library with the new one, similar to the PR I've opened in kubescape/kubescape.

Thoughts?

Reply:
SGTM